### PR TITLE
Removed event handler from exporter

### LINKED ISSUE #291: Exporter does not need to handle events
As of the latest changes to cilantro, the exporter here:
https://github.com/cbmi/varify/blob/master/varify/static/js/src/ui/exporter.js
does not need to handle events as the exporter.js file in cilantro takes care of this.

Note however, that varify must be upgraded to the latest release of cilantro before changing the native exporter.js file.

### PR DESCRIPTION
**Pull request by hassanNS**
Fix #291 
Please upgrade to the latest version of cilantro before merging this PR.
Signed-off-by: Sheik Hassan solergiga@yahoo.com

### CODE REVIEW

**Comment by naegelyd** [comment]
This cannot be removed until the new version of cilantro is included in Varify otherwise the user would not be able to change the columns.

**Comment by hassanNS** [comment]
yeah you can merge it after the upgrade

File: varify/static/js/src/ui/exporter.js [hunk 1]
```diff
@@ -7,16 +7,6 @@ define([
 
     var ExporterDialog = c.ui.ExporterDialog.extend({
         template: 'varify/export/dialog',
```
Comment (naegelyd) [inline varify/static/js/src/ui/exporter.js]: Remove trailing comma

File: varify/static/js/src/ui/exporter.js [hunk 2]
```diff
@@ -6,17 +6,7 @@ define([
 ], function(_, c) {
```
Comment (naegelyd) [inline varify/static/js/src/ui/exporter.js]: Remove underscore import, it is no longer neeeded

**Pull request merged**